src/modules/users/user.stub.ts

```diff
@@ -0,0 +1,10 @@
+import { User } from './entities/user.entity';
```

Comment:
Why need this stub in users module ?

Reply:
It was for one sample test cases for checking.